docker-for-mac/index.md

```diff
@@ -252,9 +252,7 @@ changes, click another preference tab, then choose to discard or not apply chang
 
 ### Kubernetes
 
-Docker for Mac 17.12 CE (and higher) includes a standalone Kubernetes server
-that runs on your Mac, so that you can test deploying your Docker workloads on
-Kubernetes.
+In Docker for Mac [17.12 Edge (mac45)](/docker-for-mac/edge-relese-notes/#docker-community-edition-17120-ce-mac45-2018-01-05) and higher, and [18.06 Stable (mac70)](/docker-for-mac/release-notes/#docker-community-edition-18060-ce-mac70-2018-07-25) and higher, a standalone Kubernetes server is included that runs on your Mac, so that you can test deploying your Docker workloads on Kubernetes.
```

Comment:
I'll just do the wrapping here and after a PR about the current versions can be done later on (i guess it would be suitable before the next stable desktop)